src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -518,6 +520,22 @@ ILoggingService InitializeLoggingService()
                 return loggingService;
             }
 
+            // VS builds discard many msbuild events so attach a binlogger to capture them all.
+            IEnumerable<ILogger> GetDebuggingLoggers()
```

Comment:
cc @drewnoakes on this since it interacts with project-system-tools

Reply:
@rainersigwald do you have any expectation of consequence here, or is this just an FYI?

Reply:
Just an FYI. We should happily handle having multiple loggers attached and everything should continue working as it is, but these logs might be helpful in some cases.